cg/models/demultiplex/flow_cell.py

```diff
@@ -15,20 +12,25 @@
 )
 from cg.apps.demultiplex.sample_sheet.read_sample_sheet import get_sample_sheet_from_file
 from cg.constants.constants import LENGTH_LONG_DATE
-from cg.constants.demultiplexing import (
-    BclConverter,
-    DemultiplexingDirsAndFiles,
-)
+from cg.constants.demultiplexing import BclConverter, DemultiplexingDirsAndFiles
 from cg.constants.sequencing import Sequencers, sequencer_types
 from cg.exc import FlowCellError, SampleSheetError
 from cg.models.demultiplex.run_parameters import (
     RunParameters,
     RunParametersNovaSeq6000,
     RunParametersNovaSeqX,
 )
+from pydantic.v1 import ValidationError
+from typing_extensions import Literal
 
 LOG = logging.getLogger(__name__)
 
+SampleSheetHeader = {
```

Comment:
I think constants are usually written in capital letters. We have not discussed that yet in the coding guidelines, so feel free to disregard

Reply:
Could we move the constant here: cg/constants/bcl_convert_metrics.py

Make it upper case.

Dict is fine.

Does it make sense to try and reuse some attributes from the other classes to build the headers? Feel free to disregard might become to complicated and verbose

Reply:
It makes sense, specially after moving the constant to the place you suggested. There, all the headers are already defined.

Reply:
They are not really defined there but I think I found the correct location to reuse from now :)